docs/index.md

```diff
@@ -0,0 +1,125 @@
+---
+search:
+    boost: 2
+---
+# Home
+<div style="text-align: center;">
+    <img src="assets/libdebug_logo_horiz_light.webp#only-light" loading="lazy" width="512" />
+    <img src="assets/libdebug_logo_horiz_dark.webp#only-dark" loading="lazy" width="512" />
+</div>
+
+---
+
+[![Static Badge](https://img.shields.io/badge/libdebug.org--white?style=for-the-badge&labelColor=%23009944&color=teal&link=https%3B%2F%2Flibdebug.org)](https://libdebug.org)&nbsp;&nbsp;&nbsp;
+[![DOI](https://img.shields.io/badge/DOI-10.5281%2Fzenodo.13151549-blue?style=for-the-badge)](https://doi.org/10.5281/zenodo.13151549)&nbsp;&nbsp;&nbsp;
+[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)
```

Comment:
I put it there as a placeholder because I did't like having only one or two badges, but three is a magic number and fills the bar of badges nicely. We could find a suitable alternative.

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)